src/DynamoCoreWpf/Controls/TooltipWindow.xaml

```diff
@@ -21,6 +21,155 @@
             <controls:BooleanToBrushConverter x:Key="DescriptionToColorConverter"
                                               TrueBrush="#cccccc"
                                               FalseBrush="#666666" />
+
```

Comment:
I'm just curious, is there a way to get these styles from other places like `DynamoModern.xaml`? If we can, then both `LibrarySearchView.xaml` and `LibraryView.xaml` should be using a centralized one. If we can't use the one in `DynamoModern.xaml`, then maybe at least we can make a new one just to share between both `LibrarySearchView.xaml` and `LibraryView.xaml`. Please let me know if that's possible :)